### PR TITLE
FIX doc regarding autocast feature

### PR DESCRIPTION
**Pull request by fgalan**
Based on recent dicussion on issue https://github.com/telefonicaid/iotagent-ul/issues/271

@dcalvoalonso could you have a look and review, please? Thx!

### CODE REVIEW

File: docs/usermanual.md [hunk 1]
```diff
@@ -93,6 +93,16 @@ then the NGSIv2 update uses ```10```(number), ```true``` (boolean) and ```78.8``
 
 This functionality relies on string measures casting feature implemented in the iotagent library. In order to use it, the `autocast` configuration parameter has to be set to true. Please see [configuration section of iotagent library](https://github.com/telefonicaid/iotagent-node-lib/blob/master/doc/installationguide.md#global-configuration) for further information.
 
+In addition, the device has to be provisioned using the right types for the attributes to be casted, which are:
+
+* Type "Number" for integer or float numbers
+* Type "Boolean" for boolean
+* Type "None" for null
+
+As a consecuence of the above, note the casting to JSON native format doesn't work for autoprovisioned devices as
```
Comment (dcalvoalonso) [inline docs/usermanual.md]: Small typo: As a **consequence**

File: docs/usermanual.md [hunk 2]
```diff
@@ -93,6 +93,16 @@ then the NGSIv2 update uses ```10```(number), ```true``` (boolean) and ```78.8``
 
 This functionality relies on string measures casting feature implemented in the iotagent library. In order to use it, the `autocast` configuration parameter has to be set to true. Please see [configuration section of iotagent library](https://github.com/telefonicaid/iotagent-node-lib/blob/master/doc/installationguide.md#global-configuration) for further information.
 
+In addition, the device has to be provisioned using the right types for the attributes to be casted, which are:
```
Comment (dcalvoalonso) [inline docs/usermanual.md]: to be **cast**, which are:

File: docs/usermanual.md (hunk 1 above)
Comment (fgalan) [reply docs/usermanual.md]: Fixed in a872b2ee791347f4561968624c53dca4b2495c11

File: docs/usermanual.md (hunk 2 above)
Comment (fgalan) [reply docs/usermanual.md]: Fixed in a872b2ee791347f4561968624c53dca4b2495c11

**Comment by fgalan** [comment]
Thanks for the feedback! Typos should have been fixed now.

**Comment by dcalvoalonso** [comment]
LGTM

**Pull request merged**